### PR TITLE
Editorial: use the term "IP geolocation"

### PR DESCRIPTION
**Pull request by miketaylr**
(This is more consistent with rfc 8805)

### CODE REVIEW

**Comment by miketaylr** [comment]
@DavidSchinazi PTAL?

File: Explainer-IP-Geolocation.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
 
-It’s worth noting that GeoIP —with or without IP Protection— only provides approximate and coarse location information and that there are ways in which users can obfuscate this data. With this proposal, we aim to improve user privacy while maintaining most of the existing uses of GeoIP as a coarse location signal. Addressing pre-existing accuracy issues of GeoIP is not a goal and generally, this proposal will inherit the prior limitations of IP as a source of location information.
+It’s worth noting that IP geolocatin—with or without IP Protection—only provides approximate and coarse location information and that there are ways in which users can obfuscate this data. With this proposal, we aim to improve user privacy while maintaining most of the existing uses of IP geolocation as a coarse location signal. Addressing pre-existing accuracy issues of IP geolocation is not a goal and generally, this proposal will inherit the prior limitations of IP as a source of location information.
```
Comment (DavidSchinazi) [inline Explainer-IP-Geolocation.md:7]: typo
```suggestion
It’s worth noting that IP geolocation—with or without IP Protection—only provides approximate and coarse location information and that there are ways in which users can obfuscate this data. With this proposal, we aim to improve user privacy while maintaining most of the existing uses of IP geolocation as a coarse location signal. Addressing pre-existing accuracy issues of IP geolocation is not a goal and generally, this proposal will inherit the prior limitations of IP as a source of location information.
```

**Review by DavidSchinazi — APPROVED** [review]
LGTM, thx!

File: Explainer-IP-Geolocation.md (hunk 1 above)
Comment (miketaylr) [reply Explainer-IP-Geolocation.md]: nice catch!

**Comment by miketaylr** [comment]
Thanks!

**Review by Goodness73 — COMMENTED** [review]
I want everything to work properly

**Pull request merged**